test_util/cluster.py

```diff
@@ -372,6 +371,11 @@ def wait_for_mesos_metric(cluster, host, key, value):
             add_config_path=add_config_path,
             rexray_config_preset=cluster.rexray_config_preset,
         )
+
+        logging.info("Generating node upgrade script")
+        # Generate node upgrade script
```

Comment:
nit: This comment doesn't add much value. Pretty clear that an upgrade script is being generated here.